docs/docs/docs/standalone-cli.mdx

```diff
@@ -0,0 +1,25 @@
+import { PackageManagerTabs, Badge } from '@theme';
+
+# Standalone CLI
+
+## Installation
+
+To get started, install the package using your preferred package manager:
+
+<PackageManagerTabs command="install license-kit" />
+
+## Programmatic usage (API)
+
+Once the setup is complete, you can easily add a button to your app to display the list of open-source licenses:
+
+```tsx
+
```

Comment:
probably should not be empty

Reply:
Indeed, this is still TBD - I can handle this on wednesday